sdk/compiler/damlc/tests/src/DA/Test/DamlcUpgrades.hs

```diff
@@ -411,6 +418,27 @@ tests damlc =
                       NoDependencies
                       False
                       setUpgradeField
+                , test
```

Comment:
There are two test cases in the participant you may want to reuse here: FailsWhenUpgradedFieldPackagesAreNotUpgradable and FailsWhenUpgradedFieldFromDifferentPackageName.

Reply:
Done